src/doc/rustdoc/src/documentation-tests.md

```diff
@@ -0,0 +1,3 @@
+# Documentation tests
+
+Coming soon!
```

Comment:
I am happy shipping this at whatever point, this chapter should be easy to knock out though